drivers/flash/flash_mcux_flexspi_nor.c

```diff
@@ -1093,6 +1093,36 @@ static int flash_flexspi_nor_check_jedec(struct flash_flexspi_nor_data *data,
 				kFLEXSPI_Command_READ_SDR, kFLEXSPI_1PAD, 0x01);
 		/* Device uses bit 6 of status reg 1 for QE */
 		return flash_flexspi_nor_quad_enable(data, flexspi_lut, JESD216_DW15_QER_VAL_S1B6);
+	case 0xbb20:
```

Comment:
Just FYI- I have this PR open: #82448. If this merges first I can update the value here- I would expect it to be `0x18bb20`, but could you verify that on your end?

Reply:
Ours is `0x21bb20`. So I'm not sure

Reply:
Makes sense. So I guess we could add `0x19` to `0x22`?

Reply:
Yes, we should be able to add them all to the switch statement pretty safely :)

Reply:
Note, NNba20 and NNbb20 are the same devices 0xba and 0xbb are voltages for these specific devices.
NN == 0x18 means that you have 128Mbit device, which works with 24 bit addressing everything above 0x18 will require 4 byte.
Although, I do not see any comments that the 0x18 type devices would not work with 4 byte addressing.

Reply:
@de-nordic Meaning, we should add a `case 0xba20:` there as well?

Reply:
You can, because both should be covered.

Reply:
Fixed